packages/writing-style/styles/commercetools/DateFormat.yml

```diff
@@ -7,5 +7,5 @@ ignorecase: true
 level: error
 nonword: true
 tokens:
-  - '\d{1,2}(?:\.|\/)\d{1,2}(?:\.|\/)\d{2,4}'
+  - '[^v]\d{1,2}(?:\.|\/)\d{1,2}(?:\.|\/)\d{2,4}'
```

Comment:
This is a similar case but can stay since it catches a case that can happen e.g. in SDK documetnation too.